src/material/core/_core.scss

```diff
@@ -29,7 +29,7 @@
 // pseudo element within is positioned relative to the host. Private mixin included within
 // `_mat-mdc-core`.
 @mixin _mdc-strong-focus-indicators-positioning() {
-  .mat-mdc-focus-indicator {
+  .mat-focus-indicator {
```

Comment:
I think that this entire mixin can be removed plus the `_mdc-core` above.

Reply:
Removed both mixins, thanks!